charts/orchestrator/templates/sonata-network-policy.yaml

```diff
@@ -0,0 +1,34 @@
+kind: NetworkPolicy
+apiVersion: networking.k8s.io/v1
+metadata:
+  name: RHDH-to-sonata-runtimes
```

Comment:
Can we keep this in lowercase and also use the full name "sonataflow" 
or simply `rhdh-to-sonataflow-workflows` (runtimes is this context is more of an internal term of the workflows)

Reply:
ack